dev/create-release/release-build.sh

```diff
@@ -227,7 +227,7 @@ git clean -d -f -x
 rm -f .gitignore
 cd ..
 
-export MAVEN_OPTS="-Xss128m -Xmx12g -XX:ReservedCodeCacheSize=1g"
+export MAVEN_OPTS="-Xss128m -Xmx${MAVEN_MXM_OPT:-12g} -XX:ReservedCodeCacheSize=1g"
```

Comment:
Ok, I previously changed the -Xmx option to 3g  in `build/mvn` script, not here, so the previous fix probably didn't take effect...

Reply:
I suspect so.

Reply:
> So perhaps the -Xmx here could be larger.

Why so? Do you refer to the `12g` in this line or the `2g` in the workflow? Since the build workflow uses `2g` already, we should stick to those `2g` for consistency.

Reply:
What I mean is perhaps `MAVEN_MXM_OPT` could be 4g, because I've tested before and `java-other-versions` can also run successfully on GA with 4g.

But 2g is also fine to me.

Reply:
Understood, either of `2g`, `3g`, and `4g` is fine with me, too.